src/Sylius/Bundle/CoreBundle/Announcer/BatchedVariantsUpdateAnnouncerInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Announcer;
+
+interface BatchedVariantsUpdateAnnouncerInterface
+{
+    public function dispatchVariantsUpdateCommand(array $variants): void;
```

Comment:
The current name is good. I wanted to propose alternative however:
```suggestion
    public function updateVariants(array $variants): void;
```